mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule EtsCleaner.MixProject do
     [
       app: :ets_cleaner,
       version: "0.1.0",
-      elixir: "~> 1.9",
+      elixir: "~> 1.7",
```

Comment:
Was something not working in 1.9 or 1.8?

Reply:
It all works, only Belfrage specifies ~> `1.7`

Reply:
Oh right ok, we should probably update that at some point